strawberry_django/filters.py

```diff
@@ -248,10 +247,12 @@ def build_filter_kwargs(
             subfield_filter_kwargs, subfield_filter_methods = build_filter_kwargs(
                 cast(WithStrawberryObjectDefinition, field_value),
                 f"{path}{field_name}__",
+                is_lookup=isinstance(field_value, FilterLookup),
             )
             filter_kwargs &= subfield_filter_kwargs
             filter_methods.extend(subfield_filter_methods)
-        else:
+        elif not is_lookup or (field_value is not UNSET and field_value is not None):
+            # cannot use in set because field_value can be a list which is unhashable
```

Comment:
Not sure about this... Specially after looking at the PR, silently ignoring this IMO is even worse, because someone might expect this to work.

I think the error in this case is better. But, we might try to find a way to workaround this? e.g. `name: {exact: null}` can be translated to `name__isnull=True`

Reply:
that is my thought.

Reply:
Hrm, it is an option, but I would still raise an error in this case, like "exact can't be used with `null`, use `isnull` instead" or something like this

Reply:
sounds good

Reply:
I issue now a UserWarning, or should I use an DeprecationWarning and revert for exact to the old logic?

Reply:
I think this should error out, not just a warning.

Looking from the backend perspective, I understand that the user is going to be able to see the warning. But anyone using it from the frontend can get very confused when a query for null doesn't work as expected, and silently ignores the given parameter without any error

Reply:
I see your point, but what is if the user wants null values to not error (like me)? Do we want to have setting?
In this case we can handle both ways

Reply:
Or would it be better to simply deprecate this feature? This would be the easiest way.

Reply:
I think a proper deprecation is the best way. This way we don't break anything and warn the user .

Reply:
you can use: -Wdeprecation for that.

If you still want a runtime error, I will move the error logic into an extra function, so I can overwrite it in my tests